plugins/modules/cargo.py

```diff
@@ -44,6 +44,13 @@
       try to install all of them in this version.
     type: str
     required: false
+  locked:
+    description:
+      - Install with locked dependencies.
+    required: false
+    type: bool
+    default: false
+    version_added: 6.5.0
```

Comment:
The next release this can be part of is 7.5.0:
```suggestion
    version_added: 7.5.0
```